# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gzip/hotfix.patch.sparc
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gzip-1.12/lib/string.in.h.vanilla	2023-05-28 14:22:02.357161455 +0200
+++ gzip-1.12/lib/string.in.h	2023-05-28 14:22:38.576164627 +0200
@@ -579,7 +579,7 @@
 # endif
 _GL_CXXALIASWARN (strndup);
 #else
-# if __GNUC__ >= 11
+# if __GNUC__ >= 15
 /* For -Wmismatched-dealloc: Associate strndup with free or rpl_free.  */
 _GL_FUNCDECL_SYS (strndup, char *,
                   (char const *__s, size_t __n)
